cmd/der2ascii/main.go

```diff
@@ -62,6 +64,11 @@ func main() {
 		os.Exit(1)
 	}
 
+	if *pemPassword != "" && (!*isPEM && !*isPEMAll) {
```

Comment:
Nit: don't need parens here

```suggestion
	if *pemPassword != "" && !*isPEM && !*isPEMAll {
```